### ISSUE TITLE
docker setup issues

### ISSUE THREAD

**Issue by emmakat** [1337-server/automatic-ripping-machine#56]
### Description
After a few false starts, I thought I'd recap the issue I'm facing.

I've restored the machine to a fresh image, to ensure no cruft remained. 

1. I created the arm user by using the sudo groupadd arm and sudo useradd -m arm -g arm -G  cdrom steps from the regular installation process and set the password. This user UID is 1001 
2. I setup all the drives per the instructions and added to fstab
3. I went with the pre-built option instructions, and used sudo wget to copy the udev rules to to /etc/udev/rules.d and used sudo mkdir to create the /arm/scripts folder in /opt and then used sudo wget to save the docker_arm_wrapper.sh in the scripts folder.
4. Finally, I created the container per the instructions and only modified them to include the additional drives, and to reference UID 1001 instead of 1000

Result: I can open the webserver setup, and it says it successfully created all the ARM directories, and I was able to setup the admin user. It says setup of the db was successful. There is an error  [Errno 13] Permission denied: '/opt/arm/installed' and nothings happening when I insert the disc.

I've tried this so many times, I can't imagine what to try next. I'll try one more time using the steps to build my own image, but if you could clarify whether I should be starting with a clean box with only an ARM user (no other user), then from within ARM user, I'll install docker, lsscsi, - then from which directory should I begin the git clone? Where should those files be cloned to? Does it matter? should it be in opt as is done in the manual install steps? 

### Environment
[OS Distribution and version (run "cat /etc/lsb-release")]
[ARM Release Version or if cloning from git branch/commit (run "git branch" and "git log -1" to get this)]

### Log file
[Run the rip in DEBUG and drag and drop the log file onto this comment box]

[>>>>> DRAG AND DROP LOG FILE HERE <<<<<]

**Comment by 1337-server** [comment]
` [Errno 13] Permission denied: '/opt/arm/installed'` This error can be ignored, its a remnant of the v2_devel branch, normally the devel branch creates an `installed` file inside /opt/arm to stop database deletion should the user accidentally visit the /setup page again.

Have you tried monitoring `tail -f /var/log/syslog`  when you insert a disc ?

If udev rule is setup correctly it should be calling `/opt/arm/scripts/docker_arm_wrapper.sh sr[x]` and logging success/failures in the syslog.

If you're building from your own image, personally i always git clone into /opt/arm anyway. But it shouldn't matter where as long as the git repo structure stays the same.

**Comment by emmakat** [comment]
Ok thanks! I'll try again later today and report back. tail -f /var/log/syslog wasn't doing the expected, so clearly somethings not right. I'll retry it all and start with cloning into /opt/arm

**Comment by 1337-server** [comment]
If you're not getting any output in the system log its most likely a udev issue. I would double check that the udev rule is setup on the host machine.

**Comment by emmakat** [comment]
ok! I started over entirely with an arm user, and ensuring that I was git cloning into opt/arm (needed sudo for this step). What I had to do then was all the chown and chmod gymnastics to allow arm to do the necessary steps, after the clone - which I had to use sudo for everything to work. Ripping sooooo many discs now. One tiny issue which I'll try to fix and otherwise will open an issue for, is that there doesn't seem to be an .abcde.conf file in the home/arm/config directory (I looked for it, a hidden file but it's not there at all - perhaps I'm looking in the wrong place though) - I'll try adding an old version of one I used to use in another project.
